bridge/cypress/integration/sequences.spec.ts

```diff
@@ -371,5 +371,12 @@ describe('Sequences', () => {
 
       sequencePage.assertFilterItemsCount('Service', 5).clickFilterViewMore('Service').assertFilterShowMoreCount(12);
     });
+
+    it.only('should show pause icon if sequence is paused', () => {
+      sequencePage.visit('sockshop');
+      cy.byTestId('keptn-root-events-list-e28592c6-d857-44fe-aea6-e65de02929bf')
+        .find('path')
+        .should('have.attr', 'd', 'M112 64h104v384H112zM296 64h104v384H296z');
```

Comment:
I guess there isn't an easier way to check this right? I don't know if `ng-reflect-name` would be the better choice here.
Even though, could you move this to `commands.ts` of Cypress and refactor the usages to use the same command? Something like `.assertDtIcon(type: ...)` with a switch case (map type to path attribute)